dll/win32/uxtheme/stylemap.c

```diff
@@ -1005,8 +1009,75 @@ static const MSSTYLES_CLASS_MAP classWindow[] = {
     {0,0,{'\0'}}
 };
 
+#ifdef __REACTOS__
+static const MSSTYLES_CLASS_MAP classAeroWizard[] = {
+    {AW_TITLEBAR,0,L"TitleBar"},
```

Comment:
space after comma

Reply:
I was following the style of the rest of the file, which uses no spaces throughout.

Reply:
OK.